chance.js

```diff
@@ -1875,7 +1875,7 @@ options,
                         { area: '018' + this.pick(['27','37','84','97']) + ' ', sections: [6] },
                         { area: '019' + this.pick(['00','05','35','46','49','63','95']) + ' ', sections: [6] }
                     ]);
-                    phone = options.formatted ? ukNum(numPick) : ukNum(numPick).replace(' ', '', 'g');
+                    phone = options.formatted ? ukNum(numPick) : ukNum(numPick).replace(/ /g, '', );
```

Comment:
Do you think it would be better if using a pattern to just replace any whitespace? 

`ukNum(numPick).replace(/\s/g, '') `

Reply:
Both of them would work. 
From a readability perspective, I think you're right.